### PR TITLE
application pages styling

### PR DESCRIPTION
**Pull request by paolomoz**
Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/applications/coronavirus-research/covid19-company-update
- After: https://covid19-company-update-styling--moleculardevices--hlxsites.hlx.page/applications/coronavirus-research/covid19-company-update

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -1032,7 +1032,7 @@ main div.section.wave.bluegreen {
 }
 
 .hero-container + .section > .default-content-wrapper:first-child {
-  margin-top: 50px;
+  margin-top: 15px;
```
Comment (mhaack) [inline styles/styles.css:1035]: This applies to all, not just application pages. Ist that intended and save to change?

**Review by mhaack — APPROVED** [review]

File: styles/styles.css [hunk 2]
```diff
@@ -1031,14 +1031,14 @@ main div.section.wave.bluegreen {
   padding: 30px 0;
 }
 
-.hero-container + .section > .default-content-wrapper:first-child {
```
Comment (duynguyen) [inline styles/styles.css:1034]: shouldn't we keep this as is, just add the application as below?

Comment (paolomoz) [reply styles/styles.css:1034]: yes, thanks

Comment (paolomoz) [reply styles/styles.css:1034]: done

**Pull request merged**